MobileCenter/MobileCenter/Internals/Device/MSDeviceTracker.m

```diff
@@ -128,7 +128,7 @@ - (MSDevice *)device {
 - (MSDevice *)updatedDevice {
   @synchronized(self) {
     MSDevice *newDevice = [[MSDevice alloc] init];
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
```

Comment:
If we can take care of it then I can add TODO but it is Apple tvOS SDK side, not our SDK.